components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,257 @@
+#### References
+- [Creating users in ServiceNow](https://docs.servicenow.com/bundle/madrid-platform-administration/page/administer/users-and-groups/task/t_CreateAUser.html)
+- [Assigning roles in ServiceNow](https://docs.servicenow.com/bundle/madrid-platform-administration/page/administer/users-and-groups/task/t_AssignARoleToAUser.html)
+### Application properties
+The application properties can be configured by users with `admin` or `x_chef_automate.admin` roles. Select the **Chef Automate** > **Properties** menu item to navigate to the **Properties** configuration UI.
+{{< figure src="/images/automate/snow_integration_appproperties.png" alt="Chef Automate Properties">}}
+
+The application system properties are:
+- x_chef_automate.Rest.api
+- x_chef_automate.insert_manufacturer
+- x_chef_automate.insert_model
+- x_chef_automate.client_runs_per_node
+- x_chef_automate.compliance_reports_per_node
+- x_chef_automate.chef.user
+- x_chef_automate.logging.enabled
+- x_chef_automate.chef.default.status
+- x_chef_automate.logging.verbosity
+- X_chef_automate.discovery source 
+#### Property  x_chef_automate.Rest.api
+The `x_chef_automate.Rest.api` property enables the Chef Automate API from ServiceNow when Turn on REST API is set to **Yes** by default. The possible values are:
+- Yes (default)
+- No
+#### Property x_chef_automate.insert_manufacturer
+The `x_chef_automate.insert_manufacturer` property inserts the new record during the import if a model is not found in the _core\_company_ table by setting the property to **Yes**.
+#### Property x_chef_automate.insert_model
+The `x_chef_automate.insert_model` property inserts the new record during the import if a model is not found in the _cmdb\_model_ table by setting the property to **Yes**.
+#### Property x_chef_automate.client_runs_per_node
+These 2 properties are used to set a maximum number of clients runs and compliance reports for a node. The default value is **5**.
+#### Property x_chef_automate.compliance_reports_per_node
+These 2 properties are used to set a maximum number of clients runs and compliance reports for a node. The default value is **5**.
+#### Property  x_chef_automate.chef.user
+The `x_chef_automate.chef.user` property is used to set up user’s ID or sys ID for inserting the data.
+#### Property x_chef_automate. logging.enabled
+The `x_chef_automate. logging.enabled` property is used for flagging the enable or disable logging. The default value is **No**.
+#### Property x_chef_automate.chef.default.status
+The `x_chef_automate.chef.default. status` property is used to set up the status of the service record as **inserted** or **updated**.
+#### Property x_chef_automate.logging.verbosity
+The `x_chef_automate.logging.verbosity` property debugs the data in ServiceNow. The possible values are:
+- Debug
+- Warn
+- Info
+- Error
+It enables the selected logging level and is visible in logs.
```

Comment:
If we want the line 119 to not be part of the list, can give a line between them ?